src/OrbitQt/FilterPanelWidget.cpp

```diff
@@ -0,0 +1,41 @@
+// Copyright (c) 2020 The Orbit Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#include "FilterPanelWidget.h"
+
+#include <QFontMetrics>
+#include <QFrame>
+#include <QWidget>
+
+#include "orbitmainwindow.h"
+#include "ui_FilterPanelWidget.h"
+
+FilterPanelWidget::FilterPanelWidget(QWidget* parent)
+    : QFrame(parent), ui(new Ui::FilterPanelWidget) {
+  ui->setupUi(this);
+
+  QFontMetrics fm(ui->timerLabel->font());
+  int pixel_width = fm.width("w");
```

Comment:
Apparently `QFontMetrics::width` is deprecated, check Dimitry's PR here: https://github.com/google/orbit/pull/1692/files#diff-fc6728b247ca2e98948bc9b7413fabad7dd57246578557c0f663571bf133742b

Reply:
Changed to use `QFontMetrics::width`. Thanks for that!